com.civicrm.hrjobroles/CRM/Hrjobroles/BAO/HrJobRoles.php

```diff
@@ -21,7 +22,75 @@ public static function create($params) {
     return $instance;
   }
 
+  /**
+   * Get options for a given job roles field along with their database IDs.
+   *
+   * @param String $fieldName
+   *
+   * @return Array
+   */
+  public static function buildDbOptions($fieldName) {
+    $queryParam = array(1 => array($fieldName, 'String'));
+    $query = "SELECT t1.id, t1.label from civicrm_option_value t1
```

Comment:
I think we should use aliases like cpv for civicrm_option_value and cpg for civicrm_option_group. Nothing is wrong with using t1 and t2 but it will just make the code a bit more readable.